nh_clinical/tests/common/nh_clinical_test_utils.py

```diff
@@ -166,3 +198,32 @@ def copy_instance_variable_if_exists(self, caller, variable_name):
         instance_variable_value = getattr(self, variable_name, None)
         if instance_variable_value:
             setattr(caller, variable_name, instance_variable_value)
+
+    # TODO EOBS-753: Move to patient model? Is just a generally useful method.
```

Comment:
This method isn't really test specific and may be useful in general, would it be worth moving to the patient model?
You could always keep this method and just call to the implementation in the patient model for convenience.